crates/pop-cli/src/commands/up/contract.rs

```diff
@@ -194,14 +194,18 @@ impl UpContractCommand {
 				spinner.start("Uploading contract...");
 
 				if self.upload_only {
-					let result = upload_contract_signed(self.url.as_str(), payload).await;
```

Comment:
Previous versions were less code, but resulted in the need for an expect/unwrap which is eliminated by using a match.